dace/sdfg/sdfg.py

```diff
@@ -27,6 +27,7 @@
 from dace.sdfg.replace import replace, replace_properties, replace_properties_dict
 from dace.sdfg.validation import (InvalidSDFGError, validate_sdfg)
 from dace.config import Config
+from dace.data import find_new_name
```

Comment:
do not import functions or classes from other modules (we are making this a requirement for future code and have yet to clean the past code). instead, use dt.find_new_name (if you look at line 25 it is already imported)

Reply:
Done.